service/controller/resource/monitoring/remotewriteingress/v1beta1/resource.go

```diff
@@ -0,0 +1,80 @@
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/giantswarm/k8sclient/v7/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	networkingv1beta1 "k8s.io/api/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/prometheus-meta-operator/v2/service/controller/resource/generic"
+	"github.com/giantswarm/prometheus-meta-operator/v2/service/key"
+)
+
+const (
+	Name = "monitoringremotewriteingress"
+)
+
+type Config struct {
+	K8sClient  k8sclient.Interface
+	Logger     micrologger.Logger
+	BaseDomain string
+}
+
+func New(config Config) (*generic.Resource, error) {
```

Comment:
This file is same for `v1`  and `v1beta1`, can you use a generic one?

Reply:
You mean the config? I'd rather keep the 2 separate especially since I'm not sure we will keep v1beta1

Reply:
Yes!